src/CalendarHeader.tsx

```diff
@@ -10,40 +10,59 @@ interface CalendarHeaderProps<T> {
   cellHeight: number
   style: ViewStyle
   allDayEvents: Event<T>[]
+  onPressDateHeader?: (date: string) => void
```

Comment:
Can you set `date` to `Date`? Because `Calendar` receives & returns `Date` object with its parent.

Reply:
Sure, will change it in a bit.